aspnetcore/fundamentals/logging/index.md

```diff
@@ -893,9 +629,253 @@ Using a third-party framework is similar to using one of the built-in providers:
 
 For more information, see each provider's documentation. Third-party logging providers aren't supported by Microsoft.
 
+<a name="nhca"></a>
+
+## Non-host console app
+
+For an example of how to use the Generic Host in a non-web console app, see the *Program.cs* file of the [Background Tasks sample app](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/fundamentals/host/hosted-services/samples) (<xref:fundamentals/host/hosted-services>).
+
+Logging code for apps without Generic Host differs in the way [providers are added](#add-providers) and [loggers are created](#create-logs). 
+
+### Non-host console app Logging providers
+
+In a non-host console app, call the provider's `Add{provider name}` extension method while creating a `LoggerFactory`:
+
+[!code-csharp[](index/samples/3.x/LoggingConsoleApp/Program.cs?name=snippet_LoggerFactory&highlight=1,7)]
+
+### Non-host console app create logs
+
+To create logs, use an <xref:Microsoft.Extensions.Logging.ILogger%601> object. Use the `LoggerFactory` to create an `ILogger`.
+
+The following example creates a logger with `LoggingConsoleApp.Program` as the category.
+
+[!code-csharp[](index/samples/3.x/LoggingConsoleApp/Program.cs?name=snippet_LoggerFactory&highlight=10)]
+
+In the following ASP.NET examples, the logger is used to create logs with `Information` as the level. The Log *level* indicates the severity of the logged event.
+
+[!code-csharp[](index/samples/3.x/LoggingConsoleApp/Program.cs?name=snippet_LoggerFactory&highlight=11)]
+
+[Levels](#log-level) and [categories](#log-category) are explained in more detail in this article.
```

Comment:
I've suggested a change from "article" to "document" a few times now. It looks like both are used more than once, so really it's just about consistency.

```suggestion
[Levels](#log-level) and [categories](#log-category) are explained in more detail in this document.
```